@here/harp-examples/src/datasource_object-picking.ts

```diff
@@ -0,0 +1,145 @@
+ * To enable text element picking set `gatherFeatureIds: true` in
+ * [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ *
+ * Now, let's write an event that fires when the user clicks the map canvas:
+ * ```typescript
+ * [[include:datasource_object_picking_1.ts]]
+ * ```
+ *
+ * All the data handling is covered by the `handlePick` function. Here we find the
+ * intersected objects, pick the first one in the array and display its
+ * data inside the helper box
+ * ```typescript
+ * [[include:datasource_object_picking_2.ts]]
+ * ```
+ */
+
+export namespace PickingExample {
+    document.body.innerHTML += `
+        <style>
+            #mouse-picked-result{
+                position:absolute;
+                bottom:5px;
+                border-radius: 5px;
+                margin-left:10px;
+                padding: 9px 12px;
+                background: #37afaa;
+                display: inline-block;
+                visibility: hidden;
+                text-align: left;
+            }
+            #mapCanvas {
+              top: 0;
+            }
+        </style>
+
+        <pre id="mouse-picked-result"></pre>
+    `;
+
+    initializeMapView("mapCanvas").catch(err => {
+        throw err;
+    });
+
+    // snippet:datasource_object_picking_2.ts
+    const element = document.getElementById("mouse-picked-result") as HTMLPreElement;
+    let currentUserData: PickResult | undefined;
+
+    function handlePick(mapViewUsed: MapView, x: number, y: number) {
+        // get an array of intersection results from MapView
+
+        let usableIntersections = mapViewUsed
+            .intersectMapObjects(x, y)
+            .filter(item => item.userData !== undefined);
+        if (usableIntersections.length > 1) {
+            usableIntersections = usableIntersections.filter(
+                item => item.userData !== currentUserData
+            );
+        }
+
+        if (usableIntersections.length === 0) {
+            // Hide helper box
+            element.style.visibility = "hidden";
+            return;
+        }
+
+        // Get userData from the first result;
+        currentUserData = usableIntersections[0].userData;
+
+        // Show helper box
+        element.style.visibility = "visible";
+
+        // Display userData inside of helper box
+        element.innerText = JSON.stringify(currentUserData, undefined, 2);
+    }
+    // end:datasource_object_picking_2.ts
+
+    /**
+     * Creates a new MapView for the HTMLCanvasElement of the given id.
+     */
+    async function initializeMapView(id: string) {
+        const canvas = document.getElementById(id) as HTMLCanvasElement;
+        const mapView = new MapView({
+            canvas,
+            theme: "resources/berlin_tilezen_base.json",
+            enableRoadPicking: true
+        });
+
+        CopyrightElementHandler.install("copyrightNotice")
+            .attach(mapView)
+            .setDefaults([
+                {
+                    id: "openstreetmap.org",
```

Comment:
Now the copyright is not matching anymore.

Reply:
Done